src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -314,3 +373,23 @@ def __validate_cmk(encryption_key_name=None,
         else:
             if any(arg is not None for arg in [encryption_key_vault, encryption_key_version, identity_client_id]):
                 logger.warning("Removing the customer encryption key. Key vault related arguments are ignored.")
+
+
+def __validate_replication(sku=None,
+                           replica_name=None,
+                           replica_location=None,
+                           no_replica=False):
+
+    if sku.lower() == 'premium' and not no_replica:
```

Comment:
Are the new arguments supported for standard tier?

Reply:
Yes they are. You can create a replica while creating a standard tier store, although --no-replica argument when provided will be ignored.

Reply:
@avanigupta I initially had a similar thought, but since this method jointly validates 3 arguments, I was wondering which of them the validation should be added to in params.

Reply:
Not sure I follow @albertofori